cassandra-3/pom.xml

```
@@ -0,0 +1,154 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>io.debezium</groupId>
+        <artifactId>debezium-connector-reactor-cassandra</artifactId>
+        <version>1.8.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>debezium-connector-cassandra-3.11</artifactId>
+    <name>Debezium Connector for Cassandra 3</name>
+    <version>1.8.0-SNAPSHOT</version>
```

```suggestion
    <version>1.8.0-SNAPSHOT</version>
```

That's not needed, it will be inherited from the parent.